REST/src/main/resources/application.properties

```diff
@@ -1,6 +1,4 @@
 server.port=9090
 spring.rabbitmq.addresses=10.30.1.140
 spring.rabbitmq.username=admin
-spring.rabbitmq.password=Admin1234@!
-queueName.gameConfig=GameConfigurationRegistrationQueueTest
-queueName.playerConfig=PlayerRegistrationQueueTest
\ No newline at end of file
+spring.rabbitmq.password=Admin1234@!
```

Comment:
:-1: that is NOT a good practice

Reply:
We know.
Will be changed.
Trucked under #58